watchfaces/004-analog-nordic.qml

```diff
@@ -0,0 +1,324 @@
+            ctx.stroke()
+            ctx.closePath()
+        }
+    }
+
+    Canvas {
+        z: 0
+        id: hourStrokes
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+
+            ctx.lineWidth = parent.width*0.0093
+            ctx.strokeStyle = Qt.rgba(1, 1, 1, 1)
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.7)
+            ctx.shadowOffsetX = 0
+            ctx.shadowOffsetY = 0
+            ctx.shadowBlur = 2
+            ctx.translate(parent.width/2, parent.height/2)
+            for (var i=0; i < 12; i++) {
+                if ( i%3 != 0) {
+                    ctx.beginPath()
+                    ctx.moveTo(0, height*0.3)
+                    ctx.lineTo(0, height*0.42)
+                    ctx.stroke()
+                }
+                ctx.rotate(Math.PI/6)
+            }
+        }
+    }
+
+    Canvas {
+        z: 0
+        id: minuteStrokes
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+            ctx.lineWidth = parent.width*0.014
+            ctx.lineCap = "round"
+            ctx.strokeStyle = Qt.rgba(1, 1, 1, 1)
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.7)
+            ctx.shadowOffsetX = 0
+            ctx.shadowOffsetY = 0
+            ctx.shadowBlur = 2
+            ctx.translate(parent.width/2, parent.height/2)
+            for (var i=0; i < 60; i++) {
+
+                ctx.beginPath()
+                ctx.moveTo(0, height*0.46)
+                ctx.lineTo(0, height*0.461)
+                ctx.stroke()
+
+                ctx.rotate(Math.PI/30)
+            }
+        }
+    }
+
+    Canvas {
+        z: 0
+        id: numberStrokes
+        anchors.fill: parent
+        antialiasing: true
+        smooth: true
+        renderTarget: Canvas.FramebufferObject
+        property var voffset: -parent.height*0.025
+        property var hoffset: parent.height*0.0
+        onPaint: {
+            var ctx = getContext("2d")
+            ctx.fillStyle = Qt.rgba(1, 1, 1, 0.9)
+            ctx.lineWidth = parent.height*0.0124
+            ctx.font = "0 " + height*0.18 + "px FatCow"
+            ctx.textAlign = "center"
+            ctx.textBaseline = 'middle';
+            ctx.strokeStyle = Qt.rgba(0, 0, 0, 0.3)
+            ctx.translate(parent.width/2, parent.height/2)
+            for (var i=0; i < 12; i=i+3) {
+                ctx.beginPath()
+                ctx.strokeText(i != 0 ? i: 12,
+                                        Math.cos((i-3)/12 * 2 * Math.PI)*height*0.346-hoffset,
+                                        Math.sin((i-3)/12 * 2 * Math.PI)*height*0.346-voffset)
+                ctx.fillText(i != 0 ? i: 12,
+                                      Math.cos((i-3)/12 * 2 * Math.PI)*height*0.34-hoffset,
+                                      Math.sin((i-3)/12 * 2 * Math.PI)*height*0.34-voffset)
+
+                ctx.closePath()
+            }
```

Comment:
Is it the expected behavior that the four "shadows" behind those numbers all go into different directions:
https://raw.githubusercontent.com/AsteroidOS/unofficial-watchfaces/master/.thumbnails/analog-nordic.jpg ?
I tried this watchface without having the FatCow font installed and it gave a very weird feeling, but maybe it looks decent with FatCow.

Reply:
the shadow is kerned to fatcow, starting at the inside of each number going to the outside.
The effekt with fatcow applied should be that the numbers "lean" or bow to the center and leave a shadow to the outside